src/test/java/com/google/devtools/build/lib/packages/util/mock/osx_cc_toolchain_config.bzl

```diff
@@ -5262,6 +5262,30 @@ def _impl(ctx):
         ],
     )
 
+    indexstore_files_feature = feature(
+        name = "indexstore_files",
+        flag_sets = [
+            flag_set(
+                actions = [
+                    ACTION_NAMES.assemble,
+                    ACTION_NAMES.preprocess_assemble,
```

Comment:
@BalestraPatrick 
My local clang invocation shows that although clang generates .o files for assemble files (.s and .S), it doesn't generate indexstore files. It just doesn't generate the indexstore files silently. Is assembly file compilation supposed to generate the indexstore file?